x-pack/plugins/cases/server/services/user_actions/index.ts

```diff
@@ -661,4 +671,48 @@ export class CaseUserActionService {
       },
     };
   }
+
+  public async getCaseUserActionStats({ caseId }: { caseId: string }) {
+    const response = await this.context.unsecuredSavedObjectsClient.find<
+      CaseUserActionAttributesWithoutConnectorId,
+      UserActionsStatsAggsResult
+    >({
+      type: CASE_USER_ACTION_SAVED_OBJECT,
+      hasReference: { type: CASE_SAVED_OBJECT, id: caseId },
+      page: 1,
+      perPage: 1,
+      sortField: defaultSortField,
+      aggs: CaseUserActionService.buildUserActionStatsAgg(),
+    });
+
+    const result = {
+      total: response.total,
```

Comment:
Btw we can move the burden to the UI.

Reply:
I noticed that while writing the tests and it always seemed logical but I didn't actually think about it from a UI perspective.

Now thinking about it, if there are no other user actions the filter will display the `1`.

The user will see the `added description` "action".
 
<img width="899" alt="Screenshot 2023-02-01 at 11 18 03" src="https://user-images.githubusercontent.com/1533137/216015917-99aa4b08-c0f5-4f62-a71e-696a9a4646b0.png">

Internally we call it `create_case` but feels ok if we interpret it as adding the description no?

What would mean

> move the burden to the UI? 

in this case?

We could add something like `the case was created` but then `added description` would still be displayed and the total actions would be wrong because those two are the same in the BE.

Reply:
I think we should count "description" in total as long as we are showing `added description action`.

If we don't "description" in total, we should hide `added description action` in UI as well. 
So in this case filter should show 0 when case is created.

And when user edits "description" it should be included as action.

Reply:
> > What would mean
> 
> move the burden to the UI?
> 
> in this case?

I mean the UI will subtract minus 1 from the total. I think it makes more sense if the backend is truthful of the underlying data and the UI adapts accordingly to its needs.

> Internally we call it create_case but feels ok if we interpret it as adding the description no?

Yes, the UI does not render anything when it encounters the `create_case` user action. The UI prepends the description before the user actions.

> I think we should count "description" in total as long as we are showing added description action.
> If we don't count "description" in total, we should hide added description action in UI as well.

The description is part of the case and not part of the user actions. I think the `added description` label is misleading. When we filter the user activity, the description should always stay on the top and should not be hidden no matter the filtering.

Reply:
Oh I had forgotten about the `create_case` not being displayed. Would it be more confusing if we show `All 0` and yet still show the description? Maybe we should always show a minimum of `All 1` and pretend that the description is a user action even though we're getting it from the `case.description` field?

Although I guess that gets weird when we click on a different filter. Are we planning on showing the description when you switch to the `comment` and `actions` filters?

@mdefazio what do you think? Should the description always show up?

Reply:
Maybe we can merge this PR as it is to unblock @adcoelho and decide when we implement the filtering on the UI.

Reply:
Yeah that's fine with me